### ISSUE TITLE
Allow commenting out of tests in rerun file

### ISSUE THREAD

**Issue by hugorodgerbrown** [yunojuno/django-juno-testrunner#20]
Sometimes it would be really useful to be able to run a subset of the test failures by commenting out the ones you don't want to run. Could use the python `#` style.

```
myapp.tests.TestSuite.test_one
# myapp.tests.TestSuite.test_one
myapp.tests.TestSuite.test_one
```

This should probably also be related to #4

**Comment by gergelypolonkai** [comment]
As the runner itself doesn’t read this file but you have to use it with some shell magic (from [README.rst](../README.rst): `python manage.py test < $(cat test-rerun.txt)`), there’s no way to implement this via the runner module itself. You may try this:

    python manage.py test < $(grep -v '^#' test-rerun.txt)

**Comment by stevejalim** [comment]
Even if we add this as an extra line to the README, it'll help some users

**Comment by gergelypolonkai** [comment]
Yes, but as a new test run – even if the test count is reduced – will overwrite `test-rerun.txt`, the commented test cases will be lost.
